sql-statements/sql-statement-load-data.md

```diff
@@ -12,107 +12,157 @@ The `LOAD DATA` statement batch loads data into a TiDB table.
 
-You can specify how to process the data format by configuring the `Fields` and `Lines` parameters.
+### `FORMAT`
 
-- `FIELDS TERMINATED BY`: Specifies the separating character of each data.
-- `FIELDS ENCLOSED BY`: Specifies the enclosing character of each data.
-- `LINES TERMINATED BY`: Specifies the line terminator, if you want to end a line with a certain character.
+You can use the `FORMAT` parameter to specify the format of the data file. If you do not specify this parameter, you can use the format defined by `DELIMITED DATA`, which is the data format supported by MySQL `LOAD DATA`.
+
+### `Fields`, `Lines`, and `Ignore Lines`
+
+You can specify the `Fields`, `Lines`, and `Ignore Lines` parameters only when the data format is `DELIMITED DATA`.
+
+You can use the `Fields` and `Lines` parameters to specify how to handle the data format.
+
+- `FIELDS TERMINATED BY`: specifies the data delimiter.
+- `FIELDS ENCLOSED BY`: specifies the enclosing character of the data.
+- `LINES TERMINATED BY`: specifies the line terminator, if you want to end a line with a certain character.
+
+You can use `DEFINED NULL BY` to specify how NULL values are represented in the data file.
 
 Take the following data format as an example:
 
-```
+```sql
 "bob","20","street 1"\r\n
 "alice","33","street 1"\r\n
 ```
 
-If you want to extract `bob`, `20`, and `street 1`, specify the separating character as `','`, and the enclosing character as `'\"'`:
+If you want to extract `bob`, `20`, and `street 1`, specify the delimiter as `','`, and the enclosing character as `'\"'`:
 
 ```sql
 FIELDS TERMINATED BY ',' ENCLOSED BY '\"' LINES TERMINATED BY '\r\n'
 ```
 
-If you do not specify the parameters above, the imported data is processed in the following way by default:
+If the data format is `DELIMITED DATA` and you do not specify the parameters above, the imported data is processed in the following way by default:
 
 ```sql
-FIELDS TERMINATED BY '\t' ENCLOSED BY ''
-LINES TERMINATED BY '\n'
+FIELDS TERMINATED BY '\t' ENCLOSED BY '' ESCAPED BY '\\'
+LINES TERMINATED BY '\n' STARTING BY ''
 ```
 
-### `IGNORE number LINES`
+You can ignore the first `number` lines of a file by configuring the `IGNORE <number> LINES` parameter. For example, if you configure `IGNORE 1 LINES`, the first line of a file is ignored.
+
+### `WITH detached`
 
-You can ignore the first `number` lines of a file by configuring the `IGNORE number LINES` parameter. For example, if you configure `IGNORE 1 LINES`, the first line of a file is ignored.
+If you do not specify the `LOCAL` parameter, you can use `WITH detached` to make `LOAD DATA` run in the background.
+
+You can view the created jobs with [SHOW LOAD DATA](/sql-statements/sql-statement-show-load-data.md) or you can use [OPERATE LOAD DATA JOB](/sql-statements/sql- statement-operate-load-data-job.md) to cancel or delete the created jobs.
+
+### `WITH batch_size=<number>`
```

Comment:
This part should not be moved later, don't break "### `WITH detached`" section

And to mention `0` means no splitting

Reply:
added